setup.py

```diff
@@ -299,17 +300,102 @@ def get_extensions():
         elif (hasattr(torch, 'is_mlu_available') and
                 torch.is_mlu_available()) or \
                 os.getenv('FORCE_MLU', '0') == '1':
+            import re
+
```

Comment:
`re` has been imported in the head of the file.

Reply:
Fixed